internal/crypto/src/webcrypto/check_certificate_trust.rs

```diff
@@ -0,0 +1,286 @@
+// Copyright 2022 Adobe. All rights reserved.
+// This file is licensed to you under the Apache License,
+// Version 2.0 (http://www.apache.org/licenses/LICENSE-2.0)
+// or the MIT license (http://opensource.org/licenses/MIT),
+// at your option.
+
+// Unless required by applicable law or agreed to in writing,
+// this software is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR REPRESENTATIONS OF ANY KIND, either express or
+// implied. See the LICENSE-MIT and LICENSE-APACHE files for the
+// specific language governing permissions and limitations under
+// each license.
+
+use asn1_rs::{Any, Class, FromDer, Header, Tag};
+use nom::AsBytes;
+use x509_parser::{
+    certificate::X509Certificate, der_parser::oid, oid_registry::Oid, x509::AlgorithmIdentifier,
+};
+
+use crate::{
+    cose::{CertificateTrustError, CertificateTrustPolicy},
+    p1363::der_to_p1363,
+    raw_signature::RawSignatureValidationError,
+    webcrypto::async_validator_for_signing_alg,
+    SigningAlg,
+};
+
+pub(crate) async fn check_certificate_trust(
+    ctp: &CertificateTrustPolicy,
+    chain_der: &[Vec<u8>],
+    cert_der: &[u8],
+    _signing_time_epoch: Option<i64>,
+) -> Result<(), CertificateTrustError> {
+    // TO DO: Add verification of signing time.
```

Comment:
May not be relevant since we do this in `check_certificate_profile` above,

Reply:
I think we can remove this comment since we check signing cert in cose_sign